fakturogen-comarch-connector/src/main/java/pl/fakturogen/comarch/connector/model/ApiItem.java

```diff
@@ -0,0 +1,116 @@
+package pl.fakturogen.comarch.connector.model;
```

Comment:
Please rename this to ItemComarch and move to model/comarch package, currently is in model.